src/providers/transactionManager/store.ts

```diff
@@ -0,0 +1,43 @@
+import { ExtrinsicStatus } from '@/joystream-lib'
+import { createStore } from '@/store'
+
+type TransactionSyncCallback = {
+  targetBlock: number
+  callback: () => void
+}
```

Comment:
Maybe it's just a preference but don't like how it's called, I would expect only a callback from this type. It's a little bit confusing. Also, I've noticed that you doing something like this: 
```tsx
cb.callback()
``` 
which is also in my opinion non-intuitive

Reply:
Hmm yeah I just couldn't come up with better names. Changed that now, hopefully it's better

Reply:
Yep, I think it's much better now  👍🏻